README.md

```diff
@@ -11,13 +11,13 @@ This contains:
 
 ImandraX currently supports Linux on x86_64, and macOS on aarch64 (ie ARM machines).
 
-To install the `imandrax-cli` locally, use:
+For most people, all that's needed to install `imandrax-cli` is:
```

Comment:
I like

    For stability, most users should install `imandra-cli`.

But the development text is fine.  There's a spelling error in "occasionally" below.  Also, I like how it reads as an adjective versus an adverb, but I think both are grammatically sound.

    For those okay with occasional breaking changes, the development version can be installed by exporting `VERSION=latest-main`: